cms/templates/cms/home_page.html

```diff
@@ -8,6 +8,26 @@
 
 {% block title %}{% trans "MIT MicroMasters" %}{% endblock %}
 
+{% block social %}
+  <meta property="og:site:name" content="MIT Micromasters">
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="http://micromasters.mit.edu/">
+  <meta property="og:title" content="MIT Micromasters">
+  <meta property="og:description" content="The MicroMasters is a
+    new digital credential for online learners. The MicroMasters
+    credential will be granted to learners who complete an
+    integrated set of graduate-level online courses. With the MicroMasters
+    credentials, learners can apply for an accelerated master's degree
+    program on campus, at MIT or other top Universities.">
+  <meta property="og:image" content="{% static '/static/images/lp_hero2.jpg' %}">
```

Comment:
I think we need absolute URLs here, at least from this [stackoverflow question](http://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url). Can you make these URLs absolute?